proto/storage/image.proto

```diff
@@ -79,6 +83,12 @@ message ImageScan {
 
 message ImageSignatureVerificationData {
     repeated ImageSignatureVerificationResult results = 1;
+    // The full image names that are verified (meaning, having a successful signature verification) keyed by the
+    // signature they were verified with.
+    message ImageReference {
+        repeated string names = 1;
+    }
+    map<string, ImageReference> verified_image_references     = 2;
```

Comment:
I think searching over a `map<string, object>` is unsupported, and is tricky to support. If there is a remote chance for these fields to be searchable in future, I recommend changing to array.
```
message ImageReference {
  string image_signature_integration_id;
  repeated string image_name;
}
repeated ImageReference verified_image_references;
```